tests/scm/scm-python-shared-atomspace.scm

```diff
@@ -17,7 +17,7 @@
 ; Define a python func returning a TV
 (python-eval "
 from opencog.atomspace import AtomSpace, TruthValue, types
-from opencog.utilities import get_type_ctor_atomspace
+from opencog.type_constructors import get_type_ctor_atomspace
```

Comment:
I don't get it ... if `utilities` didn't actually export `get_type_ctor_atomspace`, then is it really needed? How did this code work?

Or does python allow misc random modules to export symbols from other modules?  That seems very non-hygenic.